File: scripts/algorithms/C/Check If Two String Arrays are Equivalent/Check If Two String Arrays are Equivalent.java
// Runtime: 2 ms (Top 57.48%) | Memory: 41.5 MB (Top 82.47%)
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2)
    {
        return(String.join("", word1).equals(String.join("", word2)));
    }
}

File: scripts/algorithms/C/Check If Two String Arrays are Equivalent/Check If Two String Arrays are Equivalent.rs
// Runtime: 0 ms (Top 100.0%) | Memory: 2.30 MB (Top 17.91%)

impl Solution {
    pub fn array_strings_are_equal(word1: Vec<String>, word2: Vec<String>) -> bool {
        word1.join("") == word2.join("")
    }
}
